packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -206,35 +211,6 @@ export class TeleReporterReceiver {
     projectSuite._project = this._parseProject(project);
     for (const suite of project.suites)
       this._mergeSuiteInto(suite, projectSuite);
-
-    // Remove deleted tests when listing. Empty suites will be auto-filtered
```

Comment:
The test covering this code is failing!

Reply:
Removed `this._options.onUpdate();` from onBegin/onEnd in testSuiteUpdater but I don't quite like it. Alternatively, we could restore the results in the onBegin of the reporter created in _createReporter.

Reply:
Updated.